src/worker.js

```diff
@@ -92,6 +92,7 @@ const initServer = (app, config, callback) => {
   runAppLifecycle('initialize', config.plugins, config)
     .then(() => {
       server = app.listen(config.port, callback);
+      return server;
```

Comment:
and this is listen:

https://nodejs.org/api/http.html#http_server_listen_port_hostname_backlog_callback